manifests/m/moe/elaina/clash/nyanpasu/1.5.1/moe.elaina.clash.nyanpasu.locale.en-US.yaml

```diff
@@ -0,0 +1,31 @@
+# Created with komac v2.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+
+PackageIdentifier: moe.elaina.clash.nyanpasu
+PackageVersion: 1.5.1
+PackageLocale: en-US
+Publisher: Lib Nyanpasu Org
+PublisherUrl: https://github.com/LibNyanpasu
+Author: Lin Nyanpasu
+PackageName: Clash Nyanpasu
+PackageUrl: https://nyanpasu.elaina.moe/
+License: GPL-3.0
+LicenseUrl: https://github.com/LibNyanpasu/clash-nyanpasu/blob/main/LICENSE
+Copyright: © 2024 Clash Nyanpasu All Rights Reserved
+CopyrightUrl: https://nyanpasu.elaina.moe/
+ShortDescription: Clash Nyanpasu～(∠・ω< )⌒☆​
+Description: A Clash GUI based on tauri.
+Moniker: vscode

```

Comment:
```suggestion
# Moniker:
```
The `vscode` moniker is already in use by Visual Studio Code and cannot be used here